src/components/marker-chart/index.js

```diff
@@ -130,7 +118,7 @@ class MarkerChart extends React.PureComponent<Props> {
                 previewSelection,
                 maxViewportHeight,
                 viewportNeedsUpdate,
-                maximumZoom: this.getMaximumZoom(),
+                maximumZoom: 0, // In the marker chart, we can zoom in as much as we want.
```

Comment:
Setting this to 0 breaks things. Maybe `Number.MIN_VALUE` would work here.

STR:

* Go to marker chart
* Zoom all the way to 0
* Attempt to zoom out
* Things are stuck at 0

It might take a few tries to get all the way to 0ns rather than 0.000ns.

<img width="258" alt="image" src="https://user-images.githubusercontent.com/1588648/86591555-6c677180-bf57-11ea-89f9-cb4fa8e4b5cf.png">

Reply:
I tried very hard to get stuck but I could never be stuck.
That said I think it makes sense to make 1ns the minimum range. But I believe this should rather be done in the reducers handling the selection and the range.

Reply:
I tried handling this at the reducer level and this is very hard if we don't think more globally, and especially move to using integers more. So I'll resurrect the getMaximumZoom and calculate it from 1ns instead of interval.